### DISCUSSION TITLE
The game forfeited on time in six moves. Not sure why.

### DISCUSSION THREAD

**Discussion by ghost** [official-stockfish/Stockfish#3785] [Q&A] [answered]
Hello, everyone, I just "beat" stockfish in six moves in a five minute tournament game. It was a simple queens pawn opening, and the program just forfeited on time. I have attached the (zipped) pgn game file. I was running Stockfish 14 on Arena 3.5.1 on a 3970x Threadripper system. Somewhere in the middle of the game I pressed the infinity symbol on the menubar of Arena, thinking it would remove the time limit (give "infinite" time), but i guess its actually supposed to adjust the engine strength? I think doing so made the engine keep thinking and run out the time rather than make a last move within the 5 minutes. If so, then I would consider this a bug. Is adjusting the engine strength midgame allowed? Or perhaps I didn't install the game properly? Please let me know any thoughts on this subject, if anyone cares to comment. 
 

[i just beat stockfish i think.zip](https://github.com/official-stockfish/Stockfish/files/7524611/i.just.beat.stockfish.i.think.zip)
![Capture2](https://user-images.githubusercontent.com/94154247/141402732-012e4ef9-6165-44a6-987c-78fec0537ae6.PNG)

**Comment by niklasf** [comment]
Hi. Assuming the button started an infinite search in the middle of the game (probably by sending `go infinite`), this would be a bug or at least a quirk of Arena. The GUI (not the engine) is responsible for ending the game when a player flags and properly communicating the time control.

**Comment by ghost** [comment]
Thank you for replying. If it's a bug then it should be reproducible. If it's a quirk maybe it was a flipped bit by cosmic rays, who knows.

**Comment by ghost** [comment]
From what I gather, Arena 3.5.1 is closed source, and released in 2015. Don't know if its developers would fix this.

**Comment by ghost** [comment]
I was able to reproduce this behavior. I played a new game, using the same sequence of moves plus one more move, and after pressing the infinite engine strength button in the middle of the game, letting it think for a minute, then returning to a 5 minute game by pressing the button again, the engine forfeited on time.  


[stockfish forfeited again.zip](https://github.com/official-stockfish/Stockfish/files/7533812/stockfish.forfeited.again.zip)


![Capture 11-12-21](https://user-images.githubusercontent.com/94154247/141683206-512af4e0-44ee-4037-8ccb-f80e96d10955.PNG)

**Comment by ghost** [comment]
Finally, I was able to get it to forfeit a third time using a completely different set of moves. 

[stockfish forfeited 3.zip](https://github.com/official-stockfish/Stockfish/files/7533833/stockfish.forfeited.3.zip)

![Capture window 2 11-12-21](https://user-images.githubusercontent.com/94154247/141683954-99860fd9-6934-4e0d-b2f8-cce20d680975.PNG)

**Comment by ghost** [comment]
If I do not perform the maneuver with the engine strength infinite button then the game appears to move normally, and faster as well. So, it's some kind of bug, since my pc seems to be operating normally, and installation of the game seems ok.  I hope the developer of Arena would notice it. Peace out.

**Reply by Lolligerhans** [reply]
This seems intended behaviour, aborting the game by entering infinite analysis. Infinite refers to the time available not a parameter for the chess program.

No, I dont know how I clicked myself into this thread at this point in time.

**Comment by vondele** [comment, marked as answer]
since stockfish can't forfeit, this is a GUI issue, as mentioned previously.
